assemble/bin/accumulo-util

```diff
@@ -263,6 +264,9 @@ function main() {
     build-native)
       build_native "${@:2}"
       ;;
+    dumpzoo)
```

Comment:
Maybe make it zoodump or zoo-dump similar to how we have hadoop-jar.  Although, our commands aren't that consistently named so its no big deal, either way.

Reply:
I agree. It should probably have a `-` for consistency